### PR TITLE
Database configuration refactor

### PR DESCRIPTION
**Pull request by henare**
This refactors the way we setup the database :floppy_disk: They're all things I've found useful during real work to get an easy way to access the data :bar_chart: 

Hopefully the commits sufficiently explain the changes and rationale but let me know if not :ear:

### CODE REVIEW

File: database.rb [hunk 1]
```diff
@@ -3,9 +3,7 @@
 def database_config
   case ENV['RACK_ENV']
   when 'production'
-    ActiveRecord::ConnectionAdapters::ConnectionSpecification::ConnectionUrlResolver.new(
-      ENV['DATABASE_URL']
-    ).to_hash
+    ENV['DATABASE_URL']
```
Comment (equivalentideas) [inline database.rb]: This is great. Have you tested it works in production?

```
heroku run --app westconnex-m4-east-air-quality -e bundle exec irb
```

You could do that with irb, and running `AqmRecord.table_exists?` to test it.

Comment (equivalentideas) [reply database.rb]: I've just tested it and it works ❤️ great refactor @henare ✨

Comment (henare) [reply database.rb]: >Have you tested it works in production?

I didn't but I did test locally with:

    RACK_ENV=production DATABASE_URL="postgres://westconnex_m4east_aqm:password@localhost/westconnex_m4east_aqm_development" bundle exec dotenv ./scraper.rb

>You could do that with irb...

How does that work? Wouldn't it have been running the old code and therefore I couldn't test the change? :thinking: 

---

Thanks for the review and merge :trophy:

Comment (henare) [reply database.rb]: >How does that work?

Oh I think I know what you mean now - I could copypasta my code into `irb` running on Heroku to see if that bit of the change worked. Is that what you were thinking?

**Pull request merged**